web/src/Layout.jsx

```diff
@@ -66,6 +66,17 @@ function Layout({ MenuIcon, sectionTitle, SectionIcon, FooterActions, children }
     );
   };
 
+  const renderHeaderRightActions = () => {
```

Comment:
NOTE: I know it was me who choose those names back in the time. But they (_LeftAction_, _RightActions_) does not look good so good. In fact, _LeftAction_ is not in use yet, but it could became in something like _MainMenu_, _AppMenu_ or simply _Menu_ (have I heard _HamburguerMenu_? :stuck_out_tongue_closed_eyes:? On the other hand, _RightActions_ are more like _PageActions_ or _SectionActions_, which is not completely true for the `Overview` page which has the _Install_ action in the footer.

Handsdown, these small things need some love. The Layout in general needs more love now that we somehow introduced the concept of pages/sections for real.

Something for another PR, of course.

Reply:
Yes, we should find a better solution for the layout because, at this point, we are having some trouble with the footer. However, I would expect those problems to disappear as soon as we merge the new status API.